ui/src/app/shared/models.ts

```diff
@@ -851,12 +851,21 @@ export interface ResourceActionParam {
     default: string;
 }
 
+export interface ResourceActionParameters {
+    name: string;
+    value: string;
+}
+
 export interface ResourceAction {
     name: string;
     params: ResourceActionParam[];
     disabled: boolean;
     iconClass: string;
     displayName: string;
+    defaultValue: string;
```

Comment:
i think we don't need this spec as like above comment, we already have spec for it.